### PR TITLE
[RND-720] - Remove self-referring hreflang if only one version of the page exists

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Only include a self-referring hreflang if there is an alternate page, which is controlled in Storyblok.

## Justify why they are needed
Request from Peter
https://www.notion.so/hedviginsurance/Remove-self-referring-hreflang-if-only-one-version-of-the-page-exists-448cb3b2e885490d9a3815e4c07e1c84

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4260** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4260?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/HeadSeoInfo/HeadSeoInfo.tsx [hunk 1]
```diff
@@ -61,24 +61,33 @@ export const HeadSeoInfo = ({ story, robots }: Props) => {
...
   )
 }
 
-const AlternateLink = ({ fullSlug }: { fullSlug: string }) => {
+const AlternateLink = ({
+  fullSlug,
+  addHreflang = true,
+}: {
+  fullSlug: string
+  addHreflang?: boolean
```
Comment (alebedev) [inline apps/store/src/components/HeadSeoInfo/HeadSeoInfo.tsx:83]: Maybe just pass down `hreflang` value itself?

**Review by alebedev — APPROVED** [review]

File: apps/store/src/components/HeadSeoInfo/HeadSeoInfo.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/components/HeadSeoInfo/HeadSeoInfo.tsx:83]: thought about that, don't have a strong opinion so I can can change it

**Comment by gustaveen** [comment]
### Merge activity

* **May 14, 10:31 AM EDT**: @gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4260).

**Pull request merged**